app/src/main/java/org/rdtoolkit/processing/CloudworksApi.kt

```diff
@@ -12,11 +10,15 @@ import okhttp3.RequestBody.Companion.toRequestBody
 import org.rdtoolkit.interop.SessionToJson
 import org.rdtoolkit.support.model.session.TestSession
 import java.io.File
+import java.util.concurrent.TimeUnit
 
 
 class CloudworksApi(dns: String, val sessionId : String, val context : Context) {
     private val dns = dns.removeSuffix("/")
-    private val client = OkHttpClient()
+    private val client = OkHttpClient.Builder()
+            .readTimeout(0, TimeUnit.SECONDS)
```

Comment:
We've had some issues with unlimited timeouts hanging I/O indefinitely in ways that are hard to debug, so I think we should set these to real values. 

Our longest timeout on the CommCare side is [Two Minutes](https://github.com/dimagi/commcare-core/blob/b47f46d332abd402ba7833dd6617f03306272bf9/src/main/java/org/commcare/core/network/ModernHttpRequester.java#L40), maybe we can start there and see if anyone is still experiencing issues?

Reply:
👍